src/index.css

```diff
@@ -31,6 +31,11 @@ a.primary {
   color: #90278e;
 }
 
+#grid-container{
```

Comment:
question : should we put all of our css in one file or use feature-specific file (e.g. one css page for members, one css page for file repos ,...)? If only one file is used (like it is now), I think we should rename #grid-container with something like #members-list-grid-container or something like that..

Reply:
yes. I've put them in an component package specific css file.